File: solutions/java/src/main/java/com/epi/Parity1.java
package com.epi;

public class Parity1 {
  // @include
  public static short parity(long x) {
    short result = 0;
    while (x != 0) {
      result ^= (x & 1);
      x >>= 1;
    }
    return result;
  }
  // @exclude
}

File: solutions/python/parity1.py
# Parity1.h 848813e190b1b85a8e75107fe8513c3be38ad1a9
# @include
def parity(x):
    result = 0
    while x != 0:
        result ^= x & 1
        x >>= 1
    return result
# @exclude
